CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ Run the following command to build and install the source.
 The output will be stored in `/opt/pulumi/node_modules/@pulumi/kubernetes`.
 
 ```bash
-$ make build && make install
+$ make ensure build install
```

Comment:
@lblackstone just wanted to point out that I made this change to the documentation. LMK if you'd prefer that I revert that or elaborate on what those commands are doing.

Reply:
LGTM